pkgs/os-specific/linux/gfxtablet/default.nix

```diff
@@ -1,32 +1,28 @@
-{stdenv, fetchgit, linuxHeaders}:
-let
-  s = # Generated upstream information
-  rec {
-    version="git-2013-10-21";
-    name = "gfxtablet-uinput-driver-${version}";
-    rev = "c4e337ae0b53a8ccdfe11b904ff129714bd25ec4";
-    sha256 = "14ll9rls2mamllmpwjiv2dc6165plazv7ny9cliylafrwgb55l7p";
-    url = "https://github.com/rfc2822/GfxTablet.git";
-  };
+{stdenv, fetchurl, linuxHeaders}:
+
+stdenv.mkDerivation rec {
+  version = "1.4";
+  name = "gfxtablet-uinput-driver-${version}";
+
   buildInputs = [
     linuxHeaders
   ];
-in
-stdenv.mkDerivation {
-  inherit (s) name version;
-  inherit buildInputs;
-  src = fetchgit {
-    inherit (s) url sha256 rev;
+
+  src = fetchurl {
+    url = "https://github.com/rfc2822/GfxTablet/archive/android-app-${version}.tar.gz";
```

Comment:
Please use `fetchFromGitHub` with their generated tarballs (the hash will change whenever github decides to change compression levels and whatnot). The hash can be pre-computed with `nix-prefetch-url --unpack`.